assets/js/src/subscribers/importExport/step_results.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MailPoet from 'mailpoet';
+import _ from 'underscore';
+import ReactStringReplace from 'react-string-replace';
+
+function renderResultMessage(subscribersCount, segments, initialMessage) {
+  if (subscribersCount) {
+    let message = ReactStringReplace(initialMessage, '%1$s', () => (
+      <strong key="%1$s">{subscribersCount.toLocaleString()}</strong>
+    ));
+    message = ReactStringReplace(message, '%2$s', () => (
+      `"${segments.join('", "')}"`
+    ));
+    return (<p>{message}</p>);
+  }
+  return null;
+}
+
+function renderNoAction(createdSubscribers, updatedSubscribers) {
+  if (!createdSubscribers && !updatedSubscribers) {
+    return (<p>{MailPoet.I18n.t('importNoAction')}</p>);
+  }
+  return null;
+}
+
+function renderNoWelcomeEmail(addedToSegmentWithWelcomeNotification) {
+  if (addedToSegmentWithWelcomeNotification) {
+    return (<p>{MailPoet.I18n.t('importNoWelcomeEmail')}</p>);
+  }
+  return null;
+}
+
+function StepResults({
+  errors,
+  createdSubscribers,
+  updatedSubscribers,
+  segments,
+  addedToSegmentWithWelcomeNotification,
+  navigate,
+}) {
+  if (errors.length) {
+    MailPoet.Notice.error(_.flatten(errors));
+  }
+  return (
+    <>
+      <div className="updated">
+        {renderResultMessage(createdSubscribers, segments, MailPoet.I18n.t('subscribersCreated'))}
+        {renderResultMessage(updatedSubscribers, segments, MailPoet.I18n.t('subscribersUpdated'))}
+        {renderNoAction(createdSubscribers, updatedSubscribers)}
+        {renderNoWelcomeEmail(addedToSegmentWithWelcomeNotification)}
+      </div>
+      <button
+        type="button"
+        className="button-primary wysija"
+        onClick={() => navigate('step_method_selection', { trigger: true })}
+      >
+        {MailPoet.I18n.t('importAgain')}
+      </button>
+      &nbsp;&nbsp;
+      <button
```

Comment:
Why are you using a `<button>`  instead of `<a>` here?

Reply:
Because of accessibility. When using `a` I had to implement my own keyboard listener. This way it's already done. The link looks like a button anyway so there is no reason not to use it.